messages/runtest.md

```diff
@@ -34,6 +34,31 @@ NOTE: The testRunCoverage value (JSON and JUnit result formats) is a percentage
 
   <%= config.bin %> <%= command.id %> --test-level RunLocalTests --output-dir <path to outputdir> --target-org me@my.org
 
+- Run all tests in the org asynchronously:
+  
+  <%= config.bin %> <%= command.id %> --target-org myscratch
+
+- Run all tests synchronously; the command waits to display the test results until all tests finish:
+  
+  <%= config.bin %> <%= command.id %> --synchronous
+
+- Run specific tests using the --test-level flag:
+
+  <%= config.bin %> <%= command.id %> --test-level RunLocalTests
+
+- Run Apex tests on all the methods in the specified class; output results in Test Anything Protocol (TAP) format and request code coverage results:
```

Comment:
It looks like this doc in the CLI Command Reference does not have any "References" or "See Also" section.  Would it be possible to add one and move the reference to TAP here?  (The example that uses TAP is now in the CLI Command Reference instead of the DX Developer Guide but the link to the TAP reference is still in the DX Developer Guide.)

TAP website: https://testanything.org/

Existing reference in DX Developer Guide:
![Screenshot 2024-06-27 at 10 35 17 AM](https://github.com/salesforcecli/plugin-apex/assets/139700604/a24a437b-c988-4fcf-8268-7bfdfcc81a5a)

Reply:
I'll confirm but I don't think the command reference has a See Also section.